content/blog/getting-started-with-hpe-greenlake-api-for-backup-and-recovery.md

```diff
@@ -0,0 +1,591 @@
+
+![API to figure out the Azure storage location](/img/figure-out-storage-location.png)
+
+3. Moving forward, I composed the cloud protection store at the storage location using values from the previous response body. For this API execution, I created a request JSON body structure for `POST /backup-recovery/v1beta1/protection-stores` that contains the key-pair values from the previous API response. The below figure showed that JSON body to compose the cloud protection store that was connected to the HPE StoreOnce. 
+
+![API composing the protection store](/img/api-compose-protection-store.png)
+
+> The complete information about [this](https://developer.greenlake.hpe.com/docs/greenlake/services/backup-recovery/public/openapi/backup-recovery-public-v1beta1/operation/ProtectionStoreCreate/) JSON body structures is provided on the interactive documentation of `POST /backup-recovery/v1beta1/protection-stores` under Payload tab.
+
+![API Request body JSON to compose protection Store](/img/api-to-create-protection-stores-request-json-body.png)
+
+4. After roughly about 5 minutes, the cloud protection store was completely created based on response of the following `GET /data-services/async-operations` [API](https://developer.greenlake.hpe.com/docs/greenlake/services/data-services/public/openapi/data-services-public-v1beta1/operation/GetAsyncOperation/) as shown in the below figure. 
+
+> ***Note:*** that I used a set of selection parameters in below figure to summarize the information from this task information:
+>
+> ```shellsession
+> GET /data-services/v1beta1/async-operations/{{taskId}}/select=associatedResources,createdAt,displayName,customerId,logMessages,progressPercent,state
+> ```
+>
+> I copied the task’s id from the response header’s location value of the prior API execution into a Postman’s variable called `{{taskId}}`, and incorporated `{{taskId}}` variable to the `async-operations` API execution.
+
+![Task completion on POST protection-stores](/img/api-async-on-post-protection-stores.png)
+
+5. For reference, I went into the Protection Stores menu as shown in the below figure to show that HPE GreenLake Backup and Recovery, and confirmed that the desired protection store was available.
+
+![Validating cloud protection store has been created](/img/ui-validated-cloud-protection-store-completed.png)
+
+## Creating a Protection-Policy
+
+One of the common use-case that every user of HPE GreenLake Backup and Recovery will deploy is the creation of a protection-policy. This resource is important because it sets up the schedule for creation of recovery point; additionally, it sets up the flow of a recovery point from a primary storage to a storage snapshot, to on-premises protection-store and eventually to the cloud protection-store. 
```

Comment:
change to read "A common use-case that every..."

Break up the second sentence into two. Place a period where you have a semi-colon. Start the next sentence with "Additionally, it sets up..."